<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlutoTestControlWindow</class>
 <widget class="QMainWindow" name="PlutoTestControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>228</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>395</width>
    <height>228</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>395</width>
    <height>228</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PLUTO Test Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gbControlType">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Choose Control Type</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>347</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioNone">
           <property name="text">
            <string>No Control</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioTorque">
           <property name="text">
            <string>Torque</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioPosition">
           <property name="text">
            <string>Position</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFeedforwardTorqueValue">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>Feedforward Torque Value (Nm):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hSliderTorqTgtValue">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPositionTargetValue">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>Target Position Value (deg):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hSliderPosTgtValue">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
